examples/5_Pulse_programming_and_dynamical_decoupling.ipynb

```diff
@@ -86,7 +86,7 @@
     "\n",
     "@aq.main\n",
     "def idle():\n",
-    "    control_frame = device.frames[f\"q{qubit}_rf_frame\"]\n",
+    "    control_frame = device.frames[f\"Transmon_{qubit}_charge_tx\"]\n",
```

Comment:
The capture can only apply on readout frame, `Transmon_{qubit}_readout_rx` which is different from the charge frame. I was already incorrect before this PR but it's good to correct it :)

Reply:
Good catch! will fix this